### PR TITLE
746 - Fixed the potential issue with the Onsite logger

### PR DESCRIPTION
**Pull request by EvanParish**
#746 
Branch: bug/746
This is a quick fix to make sure we aren't getting errors when trying to report logs for Onsite notifications.

### CODE REVIEW

**Review by k-macmillan — DISMISSED** [review]

File: app/va/va_onsite/va_onsite_client.py [hunk 1]
```diff
@@ -32,12 +32,13 @@ def post_onsite_notification(self, data: dict):
             response = requests.post(url=f'{ self.url_base }/v0/onsite_notifications',
                                      data=json.dumps(data),
                                      headers=self._build_header())
+
+            self.logger.info(f'onsite_notifications POST response: status_code={response.status_code}, ' +
+                             f'json={response.json()}')
```
Comment (kalbfled) [inline app/va/va_onsite/va_onsite_client.py:37]: You don't need the plus sign.
```
>>> print("print this "
... "and this")
print this and this
```

The f-string syntax makes the code harder to follow.  I would use the C formatting syntax, which Python logging statements support:
```
self.logger.info("onsite_notifications POST response: status code=%d, json=%s", response.status, response.json())
```
I think you can use [pprint](https://docs.python.org/3.8/library/pprint.html?highlight=pprint#pprint.pprint) to format the json output.

File: app/va/va_onsite/va_onsite_client.py [hunk 2]
```diff
@@ -32,12 +32,13 @@ def post_onsite_notification(self, data: dict):
...
+
         except Exception as e:
             self.logger.exception(f'Exception in post_onsite_notification: {e}')
```
Comment (kalbfled) [inline app/va/va_onsite/va_onsite_client.py:40]: I realize you didn't change this, but the exception logger includes a stack trace.  Stating the source is superfluous.
```
self.logger.exception(e)
```

**Review by kalbfled — CHANGES_REQUESTED** [review]

File: app/va/va_onsite/va_onsite_client.py (hunk 2 above)
Comment (EvanParish) [reply app/va/va_onsite/va_onsite_client.py:40]: updated

File: app/va/va_onsite/va_onsite_client.py (hunk 1 above)
Comment (EvanParish) [reply app/va/va_onsite/va_onsite_client.py:37]: I removed the `+` but I generally prefer to use f-strings and have the variables inline. Is the format you described a better practice, or just personal preference?

Comment (kalbfled) [reply app/va/va_onsite/va_onsite_client.py]: In this instance, I think it makes the line easier to read.  Either approach works.  I only use the C formatting for logging.

Comment (EvanParish) [reply app/va/va_onsite/va_onsite_client.py]: I see what you're saying, but I don't think I'll make this change. I prefer having variables inline, it's less confusing to me.

**Review by kalbfled — APPROVED** [review]

**Review by k-macmillan — APPROVED** [review]

**Pull request merged**